src/EditorFeatures/CSharpTest/QualifyMemberAccess/QualifyMemberAccessTests.cs

```diff
@@ -198,6 +198,30 @@ void M()
 CodeStyleOptions.QualifyFieldAccess);
         }
 
+        [WorkItem(28509, "https://github.com/dotnet/roslyn/issues/28509")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)]
+        public async Task QualifyFieldAccess_InObjectInitializer()
```

Comment:
❕ This is a collection initializer, not an object initializer (test is correct, but the method name here is wrong)